components/builder-db/Cargo.toml

```diff
@@ -23,13 +23,14 @@ num_cpus = "*"
 protobuf = "*"
 fnv = "*"
 fallible-iterator = "*"
+percent-encoding = "*"
 postgres = "0.15.2"
 postgres-derive = "0.3.3"
 postgres-shared = "*"
 r2d2_postgres = "0.14.0"
 threadpool = "*"
 # Unlock with builder_core
```

Comment:
Dont need the comment any more?

Reply:
Addressed in a33367f0febdd930dfb5cba3892af203fc5b3a51